src/domain/payments/index.types.d.ts

```diff
@@ -82,7 +82,7 @@ type LightningPaymentFlowBuilder<S extends WalletCurrency> = {
     description,
   }: {
     uncheckedAmount: number
-    description: string
+    description: string | null
```

Comment:
Why did you add `| null` here?

Reply:
That was how it was before the fix. I thought that is what you wanted to see

Reply:
oh, yeah. Then this was the problem, since this is the type TS is using. The implementation type was not being picked up.

I pushed an example of how I'd use the main type in the implementation (since without it, we have implicit anys).